scripts/util.js

```diff
@@ -0,0 +1,41 @@
+const tmp = require("tmp");
+const ts = require("typescript");
+const path = require("path");
+const fs = require("fs-extra");
+
+tmp.setGracefulCleanup(); // cleans tmp file on process exit
+exports.corvidDir = path.join(__dirname, "../");
+
+const prepareEmptyTypescriptProgram = configPath => {
+  const tmpDir = tmp.dirSync();
+
+  const tmpDirPath = tmpDir.name;
+
+  const tmpConfigPath = `${tmpDirPath}/tsconfig.json`;
+  const tmpConfigContent = `{"extends": "${path.resolve(
+    exports.corvidDir,
+    configPath
+  )}"}`;
+
+  fs.writeFileSync(tmpConfigPath, tmpConfigContent);
```

Comment:
Why are you creating a new config that is just "extending" the provided config? What is the need for this? Why not just use the config provided to you?

Reply:
I think this was done to avoid having to generate a .js file to compile inside the project and to avoid compiler artifacts sticking around in the project dir